integrationtest/backend/elasticsearch/src/test/java/org/hibernate/search/integrationtest/backend/elasticsearch/management/ElasticsearchSchemaAttributeValidationIT.java

```diff
@@ -0,0 +1,582 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.integrationtest.backend.elasticsearch.management;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.hibernate.search.engine.backend.document.model.dsl.IndexSchemaElement;
+import org.hibernate.search.integrationtest.backend.elasticsearch.testsupport.util.TestElasticsearchClient;
+import org.hibernate.search.integrationtest.backend.tck.testsupport.util.rule.SearchSetupHelper;
+import org.hibernate.search.util.common.SearchException;
+import org.hibernate.search.util.impl.integrationtest.common.FailureReportUtils;
+import org.hibernate.search.util.impl.test.SubTest;
+
+import org.junit.Assume;
+import org.junit.Rule;
+import org.junit.Test;
+
+public class ElasticsearchSchemaAttributeValidationIT {
```

Comment:
Not sure why you're splitting it from the other test, since most of the validation is about attributes anyway... ?

Reply:
I would like to keep apart tests on general validation aspects, such as general attribute leniency or aggregate errors and so on, from attribute-specific validation test.